frontend/components/nav_bar/nav_bar.tsx

```diff
@@ -0,0 +1,43 @@
+import Text from "@components/text/text";
+import classNames from "classnames";
+import Image from "next/image";
+import Link from "next/link";
+import styles from "./nav_bar.module.scss";
+
+type NavBarProps = {
+  className?: string;
+};
+
+const NavBar = ({ className }: NavBarProps) => {
+  return (
+    <div className={classNames(styles.nav_bar, className)}>
+      <Image
+        className={styles.image}
+        src="/rtp_logo.png"
+        alt="rtp logo"
+        width={70}
+        height={70}
+      />
+      <Link className={styles.link} href="/about">
+        <Text.FunLarge className={styles.label}>About</Text.FunLarge>
+      </Link>
+      <Link className={styles.link} href="/news">
```

Comment:
The Link component was designed to be placed within a Text component rather than the other way around, this might alleviate styling issues e.g. with requiring another hover cursor pointer style when Link already does so

Reply:
I'm using next/link here